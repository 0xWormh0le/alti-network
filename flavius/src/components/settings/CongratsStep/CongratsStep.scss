@import 'styles/_shared.scss';

.CongratsStep {
  &__title {
    @include flex(row, flex-start, center);
    color: $TEXT_COLOR;
    margin-bottom: 1rem;
  }

  &__icon {
    @include icon();
  }
}
