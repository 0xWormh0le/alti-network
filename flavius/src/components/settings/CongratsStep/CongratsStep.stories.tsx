import React from 'react'
import { Story, Meta } from '@storybook/react/types-6-0'
import CongratsStep from './CongratsStep'

export default {
  title: 'Settings/CongratsStep',
  component: CongratsStep
} as Meta

const Template: Story<any> = (args) => <CongratsStep {...args} />

export const Default = Template.bind({})
Default.args = {}
