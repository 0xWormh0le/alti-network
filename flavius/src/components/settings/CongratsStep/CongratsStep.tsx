import React from 'react'

import Typography, { TypographyVariant } from 'components/elements/Typography'
import UI_STRINGS from 'util/ui-strings'
import verified from 'icons/verified.svg'
import './CongratsStep.scss'

const CongratsStep: React.FC<{}> = () => (
  <section className='CongratsStep TwoFactorAuthentication__step'>
    <Typography variant={TypographyVariant.H3} weight='semibold' className='CongratsStep__title'>
      <img src={verified} className='CongratsStep__icon' alt='checkmark' />
      {UI_STRINGS.SETTINGS.CONGRATS_ENROLL}
    </Typography>
    <Typography variant={TypographyVariant.BODY_LARGE}>{UI_STRINGS.SETTINGS.CHANGE_WILL_AFFECT_NEXT_TIME}</Typography>
    <Typography variant={TypographyVariant.BODY_LARGE}>{UI_STRINGS.SETTINGS.WE_WILL_ASK_CONFIRMATION}</Typography>
  </section>
)

export default CongratsStep
