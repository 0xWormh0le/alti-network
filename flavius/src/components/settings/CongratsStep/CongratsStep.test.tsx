import React from 'react'
import { render } from '@testing-library/react'
import CongratsStep from './CongratsStep'

describe('CongratsStep', () => {
  it('renders correctly', () => {
    const { container } = render(<CongratsStep />)
    expect(container).toMatchSnapshot()
  })
})
