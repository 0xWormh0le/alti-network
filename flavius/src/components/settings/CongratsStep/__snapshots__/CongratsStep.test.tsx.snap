// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CongratsStep renders correctly 1`] = `
<div>
  <section
    class="CongratsStep TwoFactorAuthentication__step"
  >
    <h3
      class="Typography Typography--h3 Typography--semibold CongratsStep__title"
    >
      <img
        alt="checkmark"
        class="CongratsStep__icon"
        src="verified.svg"
      />
      Congratulations, you are enrolled!
    </h3>
    <p
      class="Typography Typography--body-large"
    >
      The changes will take effect the next time you log in.
    </p>
    <p
      class="Typography Typography--body-large"
    >
      After that, whenever you log in to Altitude Networks app, we will ask you for a confirmation code.
    </p>
  </section>
</div>
`;
